### PR TITLE
#3491 - Ministry student account creation

### PR DESCRIPTION
**Pull request by katrina-aot-git**
Update ministry student account creation wording.
image
image

### CODE REVIEW

File: sources/packages/web/src/views/aest/student/StudentAccountApplicationsApproval.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
                   color="primary"
                   @click="createStudentAccount"
                   :disabled="notAllowed"
-                  >Create account for student</v-btn
```
Comment (dheepak-aot) [inline sources/packages/web/src/views/aest/student/StudentAccountApplicationsApproval.vue:27]: This is the text on the button, I couldn't see an AC to revise button text and would request to have the business add the AC. Let me know if I am missing something.

Comment (katrina-aot-git) [reply sources/packages/web/src/views/aest/student/StudentAccountApplicationsApproval.vue:27]: It was the same "Create account for student" text so I figured the first AC applied to both places that had that phrase on the page.

Comment (dheepak-aot) [reply sources/packages/web/src/views/aest/student/StudentAccountApplicationsApproval.vue:27]: Absolutely makes sense. But the AC doesn't seem to be specific on mentioning both places and requires to be updated.

Comment (dheepak-aot) [reply sources/packages/web/src/views/aest/student/StudentAccountApplicationsApproval.vue:27]: Got confirmation from the business on the same. Approving the PR.

**Review by dheepak-aot — APPROVED** [review]
Looks good 👍

**Pull request merged**